pytools/tag.py

```diff
@@ -378,23 +377,18 @@ class IgnoredForEqualityTag(Tag):
         }
 
 
-if sys.version_info >= (3, 7):
-    def __getattr__(name):
-        replacement_and_obj = _depr_name_to_replacement_and_obj.get(name, None)
-        if replacement_and_obj is not None:
-            replacement, obj, year = replacement_and_obj
-            from warnings import warn
-            warn(f"'arraycontext.{name}' is deprecated. "
-                    f"Use '{replacement}' instead. "
-                    f"'arraycontext.{name}' will continue to work until {year}.",
-                    DeprecationWarning, stacklevel=2)
-            return obj
-        else:
-            raise AttributeError(name)
-else:
-    TagsType = FrozenSet[Tag]
-    TagOrIterableType = ToTagSetConvertible
-    T_co = TypeVar("TaggableT", bound="Taggable")
+def __getattr__(name):
+    replacement_and_obj = _depr_name_to_replacement_and_obj.get(name, None)
+    if replacement_and_obj is not None:
+        replacement, obj, year = replacement_and_obj
+        from warnings import warn
+        warn(f"'arraycontext.{name}' is deprecated. "
```

Comment:
should this be

```suggestion
        warn(f"'pytools.tag.{name}' is deprecated. "
```

? (same below)

Reply:
Yeah, definitely! This was just copy pasted from a similar `arraycontext` change from what I recall.

Reply:
Changed in [6dd3883](https://github.com/inducer/pytools/pull/218/commits/6dd3883b1edff586e3f8b2bbd7ed06091695d987)